client/options.go

```diff
@@ -89,3 +89,9 @@ func WithDatadogTracing(opts ...dd_http.RoundTripperOption) Option {
 		)
 	}
 }
+
+func WithCustomTransport(transport http.RoundTripper) Option {
```

Comment:
This can be overwritten by OpenSencusTransport.

Is it possible to say forceHttp1(or somtehing) and change that boolean in the transport that is used?

Reply:
It's unfortunately not possible, since transport is an interface that may have different implementations.

I've updated the documentation to clarify how it interacts with `WithOpenCensusTracing`